pkg/types/validation/installconfig_test.go

```diff
@@ -470,7 +470,7 @@ func TestValidateInstallConfig(t *testing.T) {
 				}
 				return c
 			}(),
-			expectedError: `^compute\[0\]\.platform\.openstack: Invalid value: openstack\.MachinePool{FlavorName:"", RootVolume:\(\*openstack\.RootVolume\)\(nil\)}: cannot specify "openstack" for machine pool when cluster is using "aws"$`,
+			expectedError: `^compute\[0\]\.platform\.openstack: Invalid value: openstack\.MachinePool{FlavorName:"", RootVolume:\(\*openstack\.RootVolume\)\(nil\), AdditionalNetworkIDs:\[\]string\(nil\), AdditionalSecurityGroupIDs:\[\]string\(nil\)}: cannot specify "openstack" for machine pool when cluster is using "aws"$`,
```

Comment:
can you fix this, i think we 'll end up adding every time a new field. `openstack\.MachinePool{.*}`

Reply:
Done